manifests/kots-config.yaml

```diff
@@ -3,297 +3,703 @@ apiVersion: kots.io/v1beta1
+          repl{{    $_ := set $result "cartoAuth0CustomDomain" "YXV0aC5jYXJ0by5jb20=" }}
+          repl{{    $_ := set $result "launchDarklyClientSideId" "NjIxNjNmMzZkYjFmNGQxNDhmZmI3Y2Fh" }}
+          repl{{ else if eq $environment "staging" }}
+          repl{{    $_ := set $result "cartoAccApiDomain" "YWNjb3VudHMuc3RhZy5hcHAuY2FydG8uY29t" }}
+          repl{{    $_ := set $result "cartoAccGcpProjectId" "Y2FydG8tYWNjLXVzLWVhc3QxLTEtc3RhZw==" }}
+          repl{{    $_ := set $result "cartoAccGcpProjectRegion" "dXMtZWFzdDEtMS1zdGFn" }}
+          repl{{    $_ := set $result "cartoAuth0CustomDomain" "YXV0aC5zdGFnLmNhcnRvLmNvbQ==" }}
+          repl{{    $_ := set $result "launchDarklyClientSideId" "NjIxNjNmMTYyZDliOTUxNGI4MzI1NWM2" }}
+          repl{{ end }}
+          repl{{ toJson $result }}
+      - name: kotsEncryptionSecretKey
+        title: Customer Package Values
+        type: textarea
+        hidden: true
+        readonly: false
+        value: '{{repl RandomString 32 }}'
+      - name: kotsVarnishDebugSecret
+        title: Customer Package Values
+        type: textarea
+        hidden: true
+        readonly: false
+        value: '{{repl RandomString 16 }}'
+      - name: kotsJwtApiSecret
+        title: Customer Package Values
+        type: textarea
+        hidden: true
+        readonly: false
+        value: '{{repl RandomString 16 }}'
+      - name: kotsVarnishPurgeSecret
+        title: Customer Package Values
+        type: textarea
+        hidden: true
+        readonly: false
+        value: '{{repl RandomString 16 }}'
+      - name: projectId
+        title: Project Id
+        type: textarea
+        hidden: true
+        default: repl{{ fromJson (LicenseFieldValue "cartoPlatformDefaultSA") | dig "project_id" "" }}
+
+
+    # Postgresql
+    - name: MetadataDatabase
+      title: Metadata Database
+      description: |-
+        Here you should config the information to connect to your Metadata Database.
+        If you need further info, please check [docs.carto.com](https://docs.carto.com).
+      items:
+      - name: metadataDBHost
+        title: Host
+        type: text
+        required: true
+        affix: left
+      - name: metadataDBPort
+        title: Port
+        type: text
+        required: true
+        value: "5432"
+        default: "5432"
+        affix: right
+      - name: metadataDBUser
+        title: User
+        type: text
+        required: true
+        affix: left
+      - name: metadataDBPassword
+        title: Password
+        type: password
+        required: true
+        affix: right
+      - name: metadataDBName
+        title: Database Name
+        type: text
+        required: true
+        affix: left
+      - name: metadataDBCustomLabelFormat
+        type: label
+        affix: right
+      - name: metadataDBSSLMode
+        title: Enable TLS
+        type: bool
+        default: "0"
+        affix: left
+      - name: metadataDBCustomLabelFormat2
```

Comment:
If I were you, I would add a comment saying why this is there, to avoid problems in the future.